src/editors/containers/TextEditor/hooks.js

```diff
@@ -16,15 +16,24 @@ export const state = StrictDict({
   refReady: (val) => useState(val),
 });
 
-export const addImageUploadBehavior = ({ openModal, setImage }) => (editor) => {
+export const customSetup = ({ openModal, setImage }) => (editor) => {
```

Comment:
Previous pass last night, I had it named `addCustomBehavior` or something like that. I thought it would be more clear to name the hook by how it is used (passed to the `setup` field), so I changed it, but I could see going back on that...